test/helpers/file_resolver.dart

```diff
@@ -9,7 +9,13 @@ class FileResolver {
   static Future<InternalResolvedUnitResult> resolve(
     String filePath,
   ) async {
-    final path = normalize(File(filePath).absolute.path);
+    final file = File(filePath);
+
+    if (!file.existsSync()) {
+      throw StateError('Unable to find a file for the given path: $filePath');
+    }
+
+    final path = normalize(file.absolute.path);
```

Comment:
@incendial why do we need normalize absolute path?

Reply:
@dkrutskikh we have test cases where normalization is needed. So this solution works for all cases.